Header header
int32 node_id
float64 x
float64 y
float64 z
float64 qx
float64 qy
float64 qz
float64 qw